dev/source-release.sh

```diff
@@ -68,6 +69,11 @@ if [ -z "$version" ] || [ -z "$rc" ]; then
   usage
 fi
 
+if ! git ls-remote --quiet --exit-code "$remote"; then
```

Comment:
TODO - The error logs still print out, so need to redirect error stream to /dev/null.